cloudferry/config.py

```diff
@@ -38,6 +39,134 @@
 ]
 
 
+class ValidationError(Exception):
+    def __str__(self):
+        try:
+            if self.args:
+                return self.args[0] % self.args[1:]
+            else:
+                return 'ValidationError'
+        except Exception:  # pylint: disable=broad-except
```

Comment:
I'm suspect catching `TypeError` here should be enough, `Exception` is way too broad